src/uglify/index.js

```diff
@@ -1,10 +1,9 @@
 import os from 'os';
+import cacache from 'cacache';
```

Comment:
I'd suggest that if it's renamed, we don't go with `cache` as that's used all over the place in webpack already which may / may not be confusing.